static/js/apps/visualization/chart.tsx

```diff
@@ -20,14 +20,101 @@
 
 import React, { useContext } from "react";
 
+import { LineTile } from "../../components/tiles/line_tile";
+import { MapTile } from "../../components/tiles/map_tile";
+import { ScatterTile } from "../../components/tiles/scatter_tile";
+import { StatVarInfo } from "../../shared/stat_var";
+import { getStatVarGroups } from "../../utils/app/timeline_utils";
 import { isSelectionComplete } from "../../utils/app/visualization_utils";
 import { AppContext } from "./app_context";
 import { StatVarSelector } from "./stat_var_selector";
+import { VisType } from "./vis_type_configs";
 
 export function Chart(): JSX.Element {
   const { visType, places, statVars, enclosedPlaceType } =
     useContext(AppContext);
 
+  function renderChart(): JSX.Element {
+    const chartHeight = window.innerHeight * 0.45;
+    const statVarSpecs = statVars.map((sv) => {
+      return {
+        denom: "",
+        log: false,
+        name: sv.info.title || sv.dcid,
+        scaling: 1,
+        statVar: sv.dcid,
+        unit: "",
+      };
+    });
+    const statVarLabels = statVars.map((sv) => sv.info.title || sv.dcid);
+    const statVarInfo: Record<string, StatVarInfo> = {};
+    let lineChartGrouping: {
+      groups: { [groupId: string]: string[] };
+      chartOrder: string[];
+    };
+    switch (visType) {
+      case VisType.MAP:
+        return (
+          <MapTile
+            id="vis-tool-map"
+            place={places[0]}
+            statVarSpec={statVarSpecs[0]}
+            enclosedPlaceType={enclosedPlaceType}
+            svgChartHeight={chartHeight}
+            title={statVarLabels[0] + " (${date})"}
+            showLoadingSpinner={true}
+          />
+        );
+      case VisType.SCATTER:
+        return (
+          <ScatterTile
+            id="vis-tool-scatter"
+            title={
+              statVarLabels[0] +
+              " (${yDate}) vs " +
+              statVarLabels[1] +
```

Comment:
What happens if there is only one StatVar and "SCATTER" is selected as VisType?

Reply:
there is a util function `isSelectionComplete` where if the selection is not complete, the initial selection UI will be rendered instead of the charts UI